clients/python/tests/test_formatters_shared.py

```diff
@@ -80,7 +80,13 @@ def test_table_pulsar_config_html():
     namespace = "test-namespace"
     topic_name = "my-topic"
     pulsar = pulsar_pb.PulsarConfig(
-        **{"broker_service_url": broker_service_url, "auth_plugin": auth_plugin, "tenant": tenant, "namespace": namespace, "topic_name": topic_name }
+        **{
+            "broker_service_url": broker_service_url,
+            "auth_plugin": auth_plugin,
+            "tenant": tenant,
+            "namespace": namespace,
+            "topic_name": topic_name,
```

Comment:
This omits the auth fields on purpose, correct?

Reply:
this is just a reformat of an existing test.  though I assume whomever created it wanted to omit auth fields on purpose.